index.html

```diff
@@ -312,14 +312,16 @@ <h3>
 listed here.
           </p>
           <ul>
+            <li>Test suites for all normative deliverables</li>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
+            <li>Extensions for binding multilingual resources for localized user interfaces.</li>
```

Comment:
for consistency in this bullet list
```suggestion
            <li>Extensions for binding multilingual resources for localized user interfaces</li>
```